### PR TITLE
Specify  « String » as ElementTypes

### PR DESCRIPTION
**Pull request by FrankYFTang**
Change the call to CreateListFromArrayLike with  « String » as ElementTypes

Close #47

### CODE REVIEW

**Comment by FrankYFTang** [comment]
@ryzokuken @sffc @younies @zbraniecki

**Comment by FrankYFTang** [comment]
@michaelficarra @rbuckton  @rkirsling

File: spec.emu [hunk 1]
```diff
@@ -150,7 +150,7 @@ contributors: Ujjwal Sharma, Younies Mahmoud
             1. Append _field_ as the last element of _list_.
           1. Set _durationFormat_.[[Fields]] to _list_.
         1. Else if IsArray(_fields_) is *true*, then
-          1. Set _fields_ to ? CreateListFromArrayLike(_fields_).
+          1. Set _fields_ to ? CreateListFromArrayLike(_fields_, « String »).
```
Comment (rbuckton) [inline spec.emu:153]: Do you want this to produce an error if the field isn't a string, i.e `[{ toString() { return "year" } }]`, or do you want to coerce to string via `ToString`? I'm not certain if there's precedent in `Intl`, but `Object.defineProperty` uses `ToString()`. If throwing a `TypeError` for non-string values for fields is what you want, then this looks fine to me.

**Review by rbuckton — APPROVED** [review]

**Comment by FrankYFTang** [comment]
@sffc just told me after I file this issue that we are going to abandon the approach of reading an array of "fields" from the options but rather read a "smallestUnit" and a "largestUnit" instead. So I guess that mean the whole call to  CreateListFromArrayLike will be removed anyway. probably a moot point of this PR after I learn of that. Close this PR once we land that one.

**Comment by ryzokuken** [comment]
Thanks for the PR, @FrankYFTang. And yes, as @sffc said, this is not necessary since we are moving away from `"fields"` per #32  instead.

**Pull request closed without merging**